app/main.py

```diff
@@ -1,6 +1,33 @@
+import os
+
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+URL = f"http://api.weatherapi.com/v1/current.json?key={API_KEY}&q={CITY}"
```

Comment:
consider using constant url as a base url, in this example:
`http://api.weatherapi.com/v1/current.json?`
And build a full url in `get_weather()`

Reply:
I don't understand what do you want and why. You adviced to use url as constant, it is applied. This task does not require additional work with url or rebuild url to build a json with it. It is redundant feature.